src/settings_gui.cpp

```diff
@@ -323,6 +323,7 @@ struct GameOptionsWindow : Window {
 				}
 				break;
 			}
+			case WID_GO_VIDEO_DRIVER_INFO:     SetDParamStr(0, VideoDriver::GetInstance()->GetInfoString()); break;
```

Comment:
Should probably be multiple lines

Reply:
All of the above ones (before the one multiline case) are formatted the same way. If I moved it before the multiline case it would fit in?